File: 7kyu/basic_math_add_or_subtract.rb
def calculate(str)
  return str.scan(/\d+|minus\d+|plus\d+/).map {|s| s.gsub(/minus/, '-').gsub(/plus/, '')}.map(&:to_i).sum.to_s;
end
//https://www.codewars.com/kata/5809b62808ad92e31b000031

File: 7kyu/basic_math_add_or_subtract.js
var calculate = str => str.match(/\d+|minus\d+|plus\d+/g).map(s => s.replace(/minus/g, '-').replace(/plus/g, '')).map(x => parseInt(x)).reduce((a, b) => a + b)+'';
//https://www.codewars.com/kata/basic-math-add-or-subtract
